lib/src/main/java/uk/gov/di/ipv/core/library/service/SqsService.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.di.ipv.core.library.service;
+
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import com.amazonaws.services.sqs.model.SendMessageRequest;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import uk.gov.di.ipv.core.library.auditing.AuditEvent;
+import uk.gov.di.ipv.core.library.auditing.AuditEventTypes;
+import uk.gov.di.ipv.core.library.exceptions.SqsException;
+
+import java.time.Instant;
+
+public class SqsService {
```

Comment:
I think this might be better being `AuditService` or something like that?

Reply:
Yeah true, I'll change it 👍